src/Type/Definition/FieldDefinition.php

```diff
@@ -196,38 +198,24 @@ public function assertValid(Type $parentType): void
             throw new InvariantViolation("{$parentType->name}.{$this->name}: {$error->getMessage()}");
         }
 
-        Utils::invariant(
-            ! isset($this->config['isDeprecated']),
-            sprintf(
-                '%s.%s should provide "deprecationReason" instead of "isDeprecated".',
-                $parentType->name,
-                $this->name
-            )
-        );
-
         $type = $this->getType();
         if ($type instanceof WrappingType) {
             $type = $type->getWrappedType(true);
         }
 
-        Utils::invariant(
-            $type instanceof OutputType,
-            sprintf(
-                '%s.%s field type must be Output Type but got: %s',
-                $parentType->name,
-                $this->name,
-                Utils::printSafe($this->type)
-            )
-        );
-        Utils::invariant(
-            $this->resolveFn === null || is_callable($this->resolveFn),
-            sprintf(
-                '%s.%s field resolver must be a function if provided, but got: %s',
-                $parentType->name,
-                $this->name,
-                Utils::printSafe($this->resolveFn)
-            )
-        );
+        /** @var Type&NamedType $type */
+
```

Comment:
```suggestion
```

Reply:
I actually wanted this right under the previous `if` block, as that is the code that proves this type. PHPCS screwed that up.